### DISCUSSION TITLE
Jekyll Pages with defined permalinks are not appearing on my website with custom domain

### DISCUSSION THREAD

**Discussion by yoasif** [community/community#58943] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi, my pages repo is https://github.com/yoasif/yoasif.github.io

I have added some pages to my blog at /firefox/wiki/ and they appear fine locally at localhost:4000/firefox/wiki/recommended-extensions/ but return a 404 when accessed at https://www.quippd.com/firefox/wiki/recommended-extensions/

A friend tried cloning the repo and it was accessible at their github.io page.

You can look at the pages I am trying to access: https://github.com/yoasif/yoasif.github.io/tree/main/firefox/wiki if you would like to look at the code or permalink configuration. 

I looked at the build artifacts, and the pages *are* being generated in the right place. 

The files are built, per the log:

```
./firefox/
./firefox/wiki/
./firefox/wiki/recommended-extensions/
./firefox/wiki/recommended-extensions/index.html
./firefox/wiki/useful-customizations/
./firefox/wiki/useful-customizations/index.html
```

and I can see the files in the artifact from the last completed job. 

I'm out of ideas. Help!

**Comment by airtower-luna** [comment]
It seems like you have *both* a custom deployment workflow (which fails) and the default Jekyll build active in the repository. Maybe that messes things up? :thinking:

**Reply by yoasif** [reply]
The custom deployment workflow is something I tried to use when the default one didn't work for me. I went back to the default after that, as it didn't offer me an easy fix. I don't know how to remove the old jobs (or whether it matters).

**Reply by airtower-luna** [reply]
If you don't want to use the custom workflow, just remove the workflow file. I'm not sure if that'll fix things, but it removes a possible source of errors, which is always good. :wink:

**Reply by yoasif** [reply]
Okay, I went ahead and did that. It seems like that caused a rebuild. Unfortunately, the issue persists.

**Reply by airtower-luna** [reply, marked as answer]
Had a hunch and checked, here's your problem: https://github.com/yoasif/firefox

That repository has Pages active, so it gets deployed under your user domain at `/firefox/`, and that's shadowing the data you have in your user site. This happens because you added a custom domain to your user site repository, which makes it apply to *all* your repositories that don't have a custom domain of their own. :sweat_smile:

**Reply by yoasif** [reply]
Wow, this still seems like a bug to me, but I am glad to have it working now!
